client/settings/general-settings-section/index.js

```diff
@@ -3,95 +3,158 @@
  */
 import React, { useContext } from 'react';
 import styled from '@emotion/styled';
-import { __ } from '@wordpress/i18n';
-import { Card } from '@wordpress/components';
+import { Card, CheckboxControl } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
+import PaymentMethodFeesPill from 'wcstripe/components/payment-method-fees-pill';
 import CardBody from '../card-body';
-import CardsIcon from '../../payment-method-icons/cards';
-import UPEOptInBanner from '../upe-opt-in-banner';
+import UpeOptInBanner from '../upe-opt-in-banner';
 import UpeToggleContext from '../upe-toggle/context';
+import {
+	useEnabledPaymentMethods,
+	useGetAvailablePaymentMethods,
+} from './data-mock';
+import PaymentMethodDescription from './payment-method-description';
+import SectionHeading from './section-heading';
+import PaymentMethodsMap from '../../payment-methods-map';
+import PaymentMethodSetupHelp from './payment-method-setup-help';
 
-const GeneralSettingsSectionWrapper = styled.div`
-	display: flex;
-	flex-direction: column;
+const StyledUpeOptInBanner = styled( UpeOptInBanner )`
+	max-width: 100%;
+	margin: 0;
 `;
 
-const CardBodyWrapper = styled( CardBody )`
-	display: flex;
+const StyledCard = styled( Card )`
+	margin-bottom: 12px;
```

Comment:
Fixed the spacing to align with the design - notice that the design has different spacing between the list and the banner.

Before:
![Screen Shot 2021-09-07 at 5 17 41 PM](https://user-images.githubusercontent.com/273592/132556810-aa36c06d-4f00-45ae-bb74-4035f0f30de5.png)


After:
![Screen Shot 2021-09-08 at 10 58 14 AM](https://user-images.githubusercontent.com/273592/132555729-b14aca93-9a5a-4109-989c-3eca7229174f.png)